Add documentation (#8)
shotgunsoftware/tk-framework-desktopclient: docs/index.rst

The Shotgun Desktop Client Framework
==================================================

> The Shotgun Desktop Client Framework contains an implementation of a remote client that

Reviewer: ```suggestion
The Shotgun Desktop Client Framework implements a client that can communicate with Shotgun Create.
```

> has the capabilities to communicate with Shotgun Create. This client takes a Shotgun

Reviewer: ```suggestion
This client can be used to run various commands in Shotgun Create.
```

> connection and makes all operations to succeed at the Shotgun Create's desktop client handshake.

Reviewer: ```suggestion
```

.. automodule:: create_client
    :members:

.. autoclass:: CreateClient

>     :members:

Reviewer: I would like to see under this a section with the list of commands that are currently supported by Create, so our clients can know what is possible.
